### PR TITLE
Add Error Prone check: PreconditionsInvalidPlaceholder

### PR DESCRIPTION
**Pull request by ksobolew**
The format strings in preconditions only suport the `%s` placeholder.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1750,6 +1750,7 @@
                                     -Xep:ObjectToString:ERROR
                                     -Xep:OptionalEquality:ERROR
                                     -Xep:OptionalMapUnusedValue:ERROR
+                                    -Xep:PreconditionsInvalidPlaceholder:ERROR <!-- Preconditions only accepts the %s placeholder in error message strings -->
```
Comment (kokosing) [inline pom.xml:1753]: I don't think we need a comment here. It is easy to find it: https://errorprone.info/bugpattern/PreconditionsInvalidPlaceholder

**Review by kokosing — APPROVED** [review]

File: core/trino-main/src/main/java/io/trino/operator/project/MergePages.java [hunk 2]
```diff
@@ -101,7 +101,7 @@ private MergePagesTransformation(Iterable<? extends Type> types, long minPageSiz
             checkArgument(minRowCount >= 0, "minRowCount must be greater or equal than zero");
             checkArgument(maxPageSizeInBytes > 0, "maxPageSizeInBytes must be greater than zero");
             checkArgument(maxPageSizeInBytes >= minPageSizeInBytes, "maxPageSizeInBytes must be greater or equal than minPageSizeInBytes");
-            checkArgument(minPageSizeInBytes <= MAX_MIN_PAGE_SIZE, "minPageSizeInBytes must be less or equal than %d", MAX_MIN_PAGE_SIZE);
+            checkArgument(minPageSizeInBytes <= MAX_MIN_PAGE_SIZE, "minPageSizeInBytes must be less or equal than %s", MAX_MIN_PAGE_SIZE);
```
Comment (findepi) [inline core/trino-main/src/main/java/io/trino/operator/project/MergePages.java:104]: ❤️

File: pom.xml (hunk 1 above)
Comment (findepi) [reply pom.xml:1753]: agreed

**Review by findepi — APPROVED** [review]

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**